web/screenshotTesting/app.po.ts

```diff
@@ -0,0 +1,12 @@
+import { browser, by, element } from 'protractor';
```

Comment:
This directory is called `screenshotTesting` but we don't have any camel-case files or directories in our repo. Considering we have an `e2e` directory how about simply calling it `screenshot`?